generated/nifake/nifake/grpc_session_options.py

```diff
@@ -1,7 +1,14 @@
 # -*- coding: utf-8 -*-
+# This file was generated
+
 from enum import IntEnum
 
 
+# This constant specifies the gRPC package and service used by this API.
+# MeasurementLink customers should pass this value to the discovery manager to resolve the server instance that provides this interface.
+GRPC_SERVICE_INTERFACE = 'nifake_grpc.NiFake'
```

Comment:
This was the other way we were considering it:
```
class GrpcStringConstants:
  SERVICE_INSTANCE='nifake_grpc.NiFake'
  r'''Doc string'''
```
A constant of any kind in python does not support doc strings properly. Some tools like Sphinx support it though.

The [Python conventions page](https://peps.python.org/pep-0008/#constants) says the way I've got it is more desirable.

Reply:
The constant values themselves look correct. I'll let others who are more proficient with Python speak to the style/method used to expose the constants in Python.

Reply:
Does this belong in nimi-python at all? It seems backwards for these APIs to know about MeasurementLink.

Reply:
Yes Marcos it's important for those customers (the only ones we care about for gRPC-python in this release) to succeed. There's no other reasonable place for them to get this string since it may change from release to release. For example, it's possible the next release could require `nifake_grpc.NiFake-v2` and they'd have no other way of knowing it.

Reply:
FYI, a future release could also switch to a different package/service naming convention like `ni.fake.v2.NiFakeService`. 😉

https://github.com/uber/prototool/blob/dev/style/README.md#package-naming
https://docs.buf.build/best-practices/style-guide#services